tests/unit/store/traction/pacbio/runCreate/run.spec.js

```diff
@@ -246,7 +251,7 @@ describe('run.js', () => {
 
   describe('createPayload', () => {
     it('will create a new run payload', () => {
-      const run = { system_name: 'Revio' }
+      const run = { system_name: 'Revio', dna_control_complex_box_barcode: 'to keep' }
```

Comment:
I would change this system name to Sequel IIe as that is what is being selected as instrument type on line 261.